doc/man1/ts.pod

```diff
@@ -460,7 +460,7 @@ command line option. (Optional)
 =item B<signer_digest>
 
 Signing digest to use. The same as the
-B<-I<digest>> command line option. (Optional)
+B<-I<digest>> command line option. (Mandatory unless specified on command line)
```

Comment:
Why not just leave this as Optional?

Reply:
because I feel like it better reflects the reality - it needs to be specified somewhere, so specifying it is mandatory

also, specifying it in both places will give less unexpected result than not specifying it at all

i.e. "Mandatory" better manages user expectations

Reply:
grammar: specified on _the_ command line